stand/liblua/lutils.c

```diff
@@ -469,6 +484,11 @@ luaopen_loader(lua_State *L)
 	lua_add_features(L);
 	/* Set global printc to loader.printc */
 	lua_register(L, "printc", lua_printc);
+	
+	if (acpi_is_initialized()) {
+		register_lacpi_modules(L);
+	}
+
```

Comment:
to go along with the comments above, we'll need to use the gfx mechanisms and/or the kboot mechanisms so we avoid references to acpi in this file. Otherwise, other loaders will fail to load since they don't have the acpi symbols.